internal/promotion/runner/builtin/schemas/git-commit-config.json

```diff
@@ -13,20 +13,19 @@
         "email": {
           "type": "string",
           "description": "The email of the author.",
-          "oneOf": [
-            {
-              "format": "email"
-            },
-            {
-              "const": ""
-            }
-          ]
+          "format": "email"
```

Comment:
GitHub doesn't let me place this comment too far away from the nearest changed line...

Up on line 10, I believe you should probably update the description as mentioned in [this](https://github.com/akuity/kargo/pull/3922/files#r2063692442) comment.